### ISSUE TITLE
[BUG] When using resource route prefix, Show page's List button goes to /undefined 

### ISSUE THREAD

**Issue by shawngoh87** [pankod/refine#1125]
**To Reproduce**
1. Add a resource route prefix e.g. `<Resource options={{ route: "admin/posts" }}>`
2. Go to show page of that any data row
3. Click the "Posts" button to go back to list (the top right button, not sidebar)
4. The url is now at `/undefined`

**Expected behavior**
The page should go back to the list

**Desktop (please complete the following information):**
 - MacOS Catalina
 - Safari
 - Refine v1.0.79
 - Supabase data provider

**Additional context**
The Refine app lives together with another React app. All `/admin` traffic is routed to Refine app.

**Comment by omeraplak** [comment]
Hi @shawngoh87 ,
I guess I won't be able to fix this bug until routeProvider support is released. But the good news is we will release support for routeProvider soon. Please stay tuned!

**Comment by omeraplak** [comment]
> Hi @shawngoh87 , I guess I won't be able to fix this bug until routeProvider support is released. But the good news is we will release support for routeProvider soon. Please stay tuned!

But you can try this,
````
<Show
resource="posts"
...
>
```
Create, Edit, List, CreateButton, EditButton, etc. Like all components take resource prop. I think it will work for now! not a great method but it will solve your problem :)

**Comment by omeraplak** [comment]
We've released 2.0.11 with `routerProvider` support.

You can check [Serving the application from a subdirectory](https://refine.dev/docs/api-references/providers/router-provider#serving-the-application-from-a-subdirectory) doc

also [Migration guide 1.0.xx to 2.0.xx](https://refine.dev/docs/migration-guide)
